source/common/common/posix/thread_impl.h

```diff
@@ -12,16 +12,20 @@ namespace Thread {
 
 class ThreadHandle {
 public:
-  explicit ThreadHandle(std::function<void()> thread_routine);
+  ThreadHandle(std::function<void()> thread_routine, const absl::optional<int>& thread_priority);
```

Comment:
Per [abseil.io/tips/163](https://abseil.io/tips/163), this should probably just be `abls::optional<int>`. (No const, no ref)